datamodel/src/main/java/com/hartwig/serve/datamodel/serialization/util/ActionableFileUtil.java

```diff
@@ -174,6 +196,7 @@ public static String toLine(@NotNull ActionableEvent event) {
             therapy.add(treatment.name());
         }
         return new StringJoiner(FIELD_DELIMITER).add(event.source().toString())
+                .add(event.date().toString())
```

Comment:
I think we would like to keep the field empty in the tsv if the date is LocalDate.EPOCH?